rollup.config.js

```diff
@@ -1,29 +1,61 @@
+import external from 'rollup-plugin-peer-deps-external';
+import json from '@rollup/plugin-json';
 import typescript from 'rollup-plugin-typescript2';
+import commonjs from '@rollup/plugin-commonjs';
+import resolve from '@rollup/plugin-node-resolve';
+import sourcemaps from 'rollup-plugin-sourcemaps';
+import { terser } from 'rollup-plugin-terser';
+import pkg from './package.json';
 
-export function getConfig({
-  tsconfig = './tsconfig.json',
-  output = [
-    {
-      file: `dist/index.js`,
-      format: 'cjs',
-    },
-    {
-      file: `dist/index.es.js`,
-      format: 'esm',
+export default [
+  {
+    input: 'src/index.tsx',
+    output: {
+      name: 'HookFormErrorMessage',
```

Comment:
@bluebill1049 What do you think this umd global naming?
`HookFormErrorMessage` vs `ErrorMessage`

Reply:
`ReactHookFormErrorMessage` ? keep consistent?

Reply:
done!